src/schema/v2/me/artistRecommendations.ts

```diff
@@ -44,14 +55,19 @@ export const ArtistRecommendations: GraphQLFieldConfig<
           }
         }
       `,
-    })()
+    }
+
+    const isAuthenticatedRequest = !!vortexGraphQLAuthenticatedLoader
```

Comment:
I believe this will consistently fail on non-authenticated requests and the following code will be inaccessible? 🤔 

So something like can work here?

```js    
let vortexResult = null

try {
     vortexResult = await vortexGraphQLAuthenticatedLoader!(query)()
} catch (error) {
      vortexResult = await vortexGraphQLUnauthenticatedLoader(query)()
}
```